RDVFSharp/Commands/RoomCreation/RoomCreate.cs

```diff
@@ -87,10 +96,19 @@ public async new void ExecutePrivateCommand(string characterCalling, IEnumerable
 
         public override async Task ExecuteCommand(string characterCalling, IEnumerable<string> args, string channel)
         {
-            var result = await Execute(characterCalling, args);
-            foreach (var message in result)
+            if (channel == "ADH-51710b5ac8cce7e99f19" || channel == "ADH-b3c88050e9c580631c70")
```

Comment:
Same comment as before, use "public static string RDVF_VENUE_ROOMID" instead of inline strings like that